### ISSUE TITLE
How to get the classes scores used for confusion matrix with the estimator?

### ISSUE THREAD

**Issue by 1160301020** [tensorflow/models#5991]
I am currently trying to train Google's sketch recognition model，just the one in the link: https://github.com/tensorflow/models/blob/master/tutorials/rnn/quickdraw/train_model.py. But I recently encountered problems that have been bothering me for a long time. The problem is as follows: I have used the code in the link and the data from quickdraw to complete the training. I now have a trained model with three files(.meta,.index,.data), now i want to calculate the confusion matrix for the trained model of 345 categories.But since I have never used the "estimator" of tensorflow,I don't know how to load my trained model files into the code and test it (no training), and how to get the classification score after the softmax layer (used to calculate Confusion matrix). The ‘estimator’ API really confused me for a long time. I will appreciate you so much if you can help me.(Please solve my problem under the code in link)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by 1160301020** [comment]
In fact i just used the code in tensorflow official tutorial [https://github.com/tensorflow/models/blob/master/tutorials/rnn/quickdraw/train_model.py](url)
